File: Arrays&Hashing/containsDup.js
// Given an integer array nums, return true if any value appears at least twice in the array, and return false if every element is distinct.


var containsDuplicate = function(nums) {
    for(var i = 0; i < nums.length; i++) {
        for(var j = i + 1; j < nums.length; j++) {
            if (nums[i] == nums[j])
            return true;
        }
        return false;
    }
};

numss = [1,2,3,4]
nums = [1,2,3,1]

console.log(containsDuplicate(nums));
console.log(containsDuplicate(numss));

File: Arrays&Hashing/containsDup.py
def containsDuplicate(nums):
    # i to make a set
        thiset = set()
        # make a for loop
        for s in nums:
            # check in nums in set
            if s in thiset:
                # if true return True
                return True
            # if not False
            thiset.add(s)
        return False


print(containsDuplicate([1,2,3,4,5,6,7,8,9,10,11,12,13,15,15]))
print(containsDuplicate([1,2,3,4,5,6,7,8,9,10,11,12,13,14,15]))